addons/website_sale/views/templates.xml

```diff
@@ -672,6 +672,10 @@
                                 </div>
                             </div>
                             <hr />
+                            <!-- inner content is accessible for inside the product details of the shop page -->
+                            <section class="container" id="inner_content_snippets">
+                                <div class="s_wsale_products_inner_content_accessible"/>
+                            </section>
```

Comment:
dont know if we will still need this part after my change in the product page re-design where I add an option in customize show menu to display share buttons on the page product : https://i.imgur.com/EC4eUBt.jpeg
@sbuhl  @JKE-be